jetbrains-core/src/software/aws/toolkits/jetbrains/services/caws/envclient/models/UpdateActivityRequest.kt

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.caws.envclient.models
+
+data class UpdateActivityRequest(
+    val timestamp: String? = null
```

Comment:
does this actually need to be a string or will the environment API automatically accept it as a Long?

Reply:
The API takes a string value so no cast required